src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let currentState = { ...state };
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(currentState, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of action.keysToRemove) {
+          delete currentState[key];
+        }
+        break;
+
+      case 'clear':
+        currentState = {};
+        break;
+    }
```

Comment:
The switch statement should include a default case to handle any unexpected action types. This is important for error handling and to ensure that the function behaves correctly if it receives an unknown action.

Reply:
Fix this
<img width="703" alt="image" src="https://github.com/mate-academy/js_stateful-clones/assets/86789161/dad07b9e-8242-4136-9ee8-ccff528930d6">